todos/src/supabase.js

```diff
@@ -0,0 +1,17 @@
+import { createClient } from "@supabase/supabase-js";
+
+const projectURL = "https://aemdlybecnilhqobaiwn.supabase.co";
+const anonKey =
+  "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImFlbWRseWJlY25pbGhxb2JhaXduIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NzYyNjE0NDQsImV4cCI6MTk5MTgzNzQ0NH0.MawqvRVONrJWmOTjAmkSoBKLMPAl0dyiw2k2nUEt3IE";
+let supabaseInstance = null;
+
+class SupabaseSingleton {
+  constructor() {
+    if (!supabaseInstance) {
+      supabaseInstance = createClient(projectURL, anonKey);
+    }
+    return supabaseInstance;
+  }
+}
+
+export default SupabaseSingleton;
```

Comment:
We can create a the Supabase class and export it from here.
```suggestion


const supabase = new SupabaseSingleton();
export default supabase;
```